specification/common-types/resource-management/v1/types.json

```diff
@@ -399,6 +379,20 @@
         }
       }
     },
+    "createdByType": {
```

Comment:
I agree but I believe that would be considered a breaking change.  It would change the class name that contains the string constants in the .NET SDK for example.

---
In reply to: [441105069](https://github.com/Azure/azure-rest-api-specs/pull/9844#discussion_r441105069) [](ancestors = 441105069)